src/symbols.cs

```diff
@@ -0,0 +1,215 @@
+using CoreFoundation;
+using ObjCRuntime;
+using Foundation;
+
+using System;
+
+namespace Symbols {
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolEffect : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolContentTransition : NSCopying, NSSecureCoding { }
```

Comment:
Docs say that subclasses are the ones meant to be used.
```suggestion
	[BaseType (typeof (NSObject))]
	[DisableDefaultCtor]
	interface NSSymbolContentTransition : NSCopying, NSSecureCoding { }
```